src/critics/prefer_forward_critic.cpp

```diff
@@ -15,6 +16,45 @@ void PreferForwardCritic::initialize()
     logger_, "PreferForwardCritic instantiated with %d power and %f weight.", power_, weight_);
 }
 
+// Derivative of https://gist.github.com/volkansalma/2972237
+#define PI_FLOAT     3.14159265f
+#define PIBY2_FLOAT  1.5707963f
```

Comment:
constexpr

Reply:
I'm going to delete all of this since the prefer forward updates you made are better